wee_slack.py

```diff
@@ -2270,6 +2351,30 @@ def handle_history(message_json, eventrouter, **kwargs):
     for message in reversed(message_json["messages"]):
         process_message(message, eventrouter, **kwargs)
 
+
+def handle_conversationsmembers(members_json, eventrouter, **kwargs):
+    request_metadata = pickle.loads(members_json['wee_slack_request_metadata'])
+    team = eventrouter.teams[request_metadata.team_hash]
+    channel = team.channels[request_metadata.channel_identifier]
+    channel.members = set(members_json['members'])
+
+
+def handle_usersinfo(user_json, eventrouter, **kwargs):
+    request_metadata = pickle.loads(user_json['wee_slack_request_metadata'])
+    team = eventrouter.teams[request_metadata.team_hash]
+    channel = team.channels[request_metadata.channel_identifier]
+    user_info = user_json['user']
+    user_info.update(is_external=True, deleted=False)
```

Comment:
I'm not sure we should assume all `users.info` calls are for external users. Even though they are now, we might use the method for other things in the future.